### PR TITLE
[MINOR][EB-166438] Circle CI Configuration

### PR DESCRIPTION
**Pull request by vgatica-eb**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

## Description
https://eventbrite.atlassian.net/browse/EB-166438
We take this document as reference
https://docs.google.com/document/d/1alDEr8qMI99rFn0YoaDZqob9luBx8n2f8RJzTSAB8Mw/edit#

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
     executor:
       name: small-docker
     environment:
-      LIBRARY_NAME: permissions
+      LIBRARY_NAME: celery
```
Comment (gastudillo-eb) [inline .circleci/config.yml:30]: What things does this change affect?

Comment (abacigalup-eb) [reply .circleci/config.yml:30]: just checked, used for the python release process here https://github.com/eventbrite/docker-dev/blob/master/ci_cd/scripts/python/release.py#L480

Comment (abacigalup-eb) [reply .circleci/config.yml:30]: *old devpi release

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        volumes:
+            - keys:/etc/eventbrite/keys
+    keys:
+        image: 353605023268.dkr.ecr.us-east-1.amazonaws.com/keys-volume:master
+        command: ["/usr/bin/tail", "-f", "/dev/null"]
+        volumes:
+            - "keys:/etc/eventbrite/keys"
+
+volumes:
+    keys:
```
Comment (gastudillo-eb) [inline docker-compose.yml]: nl

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+      - test:
+          name: test-pull-request
+          context:
+            - 'DevTools ECR'
+            - 'Prod AWS'
+          filters:
+            branches:
+              ignore:
+                - main
+                - /^invoke-release-.*-[0-9]+\.[0-9]+\.[0-9]+\+eventbrite$/
```
Comment (gastudillo-eb) [inline .circleci/config.yml]: add nl

Comment (vgatica-eb) [reply .circleci/config.yml]: done

File: docker-compose.yml (hunk 2 above)
Comment (vgatica-eb) [reply docker-compose.yml]: done

**Review by abonilla-eb — APPROVED** [review]

**Pull request merged**